### PR TITLE
Use f-strings in examples output

### PR DESCRIPTION
**Pull request by sgerrand**
💁 As recommended by @jesse-c in #90. ([comment](https://github.com/duffelhq/duffel-api-python/pull/90#discussion_r795689602), [comment](https://github.com/duffelhq/duffel-api-python/pull/90#discussion_r795690121))

### CODE REVIEW

**Review by jesse-c — APPROVED** [review]
LGTM

The multi-line string approach looks a little un-ideal to me, but I simply may not have seen enough Python, so I'm not sure what exactly to suggest.

**Comment by sgerrand** [comment]
> The multi-line string approach looks a little un-ideal to me

Yeah, it's crap. I'm going to reverse it and use some inline flake8 directives to get around it.

**Comment by sgerrand** [comment]
I think 0754593 is a better approach.

File: examples/book-with-adult-and-infant.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
     payments = [
         {
-            "currency": selected_offer.total_currency,
+            "currency": priced_offer.total_currency,
```
Comment (sgerrand) [inline examples/book-with-adult-and-infant.py:45]: I got this wrong when I added it in #90.

File: examples/book-and-change.py [hunk 2]
```diff
@@ -20,22 +20,21 @@
...
 
-    print("Selected offer %s to book" % (selected_offer.id))
+    print(f"Selected offer {selected_offer.id} to book")
 
     priced_offer = client.offers.get(selected_offer.id)
 
     print(
-        "The final price for offer %s is %s (%s)"
-        % (priced_offer.id, priced_offer.total_amount, priced_offer.total_currency)
+        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} ({priced_offer.total_currency})"
```
Comment (richcooper95) [inline examples/book-and-change.py:37]: Not sure what it was before, but having read the comments pre-merge, thought I'd point out you can split a string across multiple lines as long as it's in parentheses (or uses a `\` at the end of the line) - i.e. this:

```python
    print(
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} ({priced_offer.total_currency})"
    )
```

could become this:

```python
    print(
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} "
        f"({priced_offer.total_currency})"
    )
```

or this, if you wanted to assign a long string to a variable (just for illustration):

```python
    msg = (
        f"The final price for offer {priced_offer.id} is {priced_offer.total_amount} "
        f"({priced_offer.total_currency})"
    ) # not a tuple!

    print(msg)
```

Note that the above doesn't construct a tuple - this is why tuples require a trailing comma when constructing via `()` syntax, e.g.

```python
>>> t = (1,)
>>> type(t)
<class 'tuple'>
>>> t = (1)
>>> type(t)
<class 'int'>
```

Comment (sgerrand) [reply examples/book-and-change.py:37]: You can, thanks for pointing this usage out, which is what I'd done. I decided that it wasn't worth the cost in readability for the adherence to PEP 8's default E501 so decided to exclude the examples from that check instead.

**Pull request merged**